.pre-commit-config.yaml

```diff
@@ -1,6 +1,9 @@
+default_language_version:
+  python: python3.10
 ci:
   autoupdate_commit_msg: "chore: pre-commit autoupdate"
   autofix_commit_msg: "chore: pre-commit auto fixes [...]"
+  skip: [poetry-lock]
```

Comment:
Just double checking, this skips running the `poetry-lock` hook, but not the action of generating a new lock file when dependabot upgrades a dependency, right?

Reply:
correct